.palantir/revapi.yml

```diff
@@ -1,4 +1,18 @@
 acceptedBreaks:
+  "1.3.0":
+    org.apache.iceberg:iceberg-api:
+    - code: "java.method.addedToInterface"
+      new: "method ThisT org.apache.iceberg.IncrementalScan<ThisT, T extends org.apache.iceberg.ScanTask, G extends org.apache.iceberg.ScanTaskGroup<T extends org.apache.iceberg.ScanTask>>::useBranch(java.lang.String)"
```

Comment:
rather than breaking the API, can we make those methods `default` and throw an `UnsupportedOperationException`?

Reply:
Yes, we could do that, but then the interfaces would seem to be optional. 
Anyway, I've made the change.